manifests/base/deployment.yaml

```diff
@@ -26,6 +26,7 @@ spec:
           - --kubelet-preferred-address-types=InternalIP,ExternalIP,Hostname
           - --kubelet-use-node-status-port
           - --metric-resolution=15s
+          - --authorization-always-allow-paths=/livez,/readyz
```

Comment:
Isn't it better to ensure those settings during start-up here https://github.com/kubernetes-sigs/metrics-server/blob/e3ed795bef95a9648a02af660dc20713faf75ac9/cmd/metrics-server/app/start.go#L47-L75 only if the k8s version is <=1.15?


It should be something like 
```golang
	kubeClient, err := kubernetes.NewForConfig(config.Rest)
	if err != nil {
		return fmt.Errorf("unable to construct kubernetes client: %v", err)
	}

	serverVersion, err := kubeClient.ServerVersion()
	if err != nil {
		return fmt.Errorf("unable to get the kubernetes server version: %v", err)
	}

	k8sVersion := semver.MustParse(serverVersion.GitVersion)
	version115, err := semver.NewConstraint("<= 1.15")
	if err != nil {
		return fmt.Errorf("unable to create version constraint: %v", err)
	}

	if version115.Check(k8sVersion) {
		o.Authorization.WithAlwaysAllowPaths("/livez", "/readyz")
	}
```